docs/src/architecture/architecture.md

```diff
@@ -29,13 +29,13 @@ discussed in Pianosi et al., 2016):
 3. Post-processing
 
 When ADRIA is applied in its entirety, "input sampling" is analogous to scenario generation: all
-model parameters (the inputs) are collated and are sampled through a quasi-monte carlo process
-and subsequently adjusted to produce plausible combinations of parameter values. The Sobol' sampling
+model factors (the inputs) are collated and are sampled through a quasi-monte carlo process
+and subsequently adjusted to produce plausible combinations of factor values. The Sobol' sampling
 method is adopted as the default, although any method provided by the [QuasiMonteCarlo.jl](https://github.com/SciML/QuasiMonteCarlo.jl)
 package can be used. Sample adjustment is required to map sampled values (which are continuous) to
 categorical or whole number values (e.g., Baroni and Tarantola, 2014) as may be expected by some
-parameters. Values are also adjusted to avoid implausible factor combinations, such as active
-intervention parameters in the case of non-intervention scenarios.
+factors. Values are also adjusted to avoid implausible factor combinations, such as active
+intervention factors in the case of non-intervention scenarios.
```

Comment:
Line 37 feels like a repeat of line 33, but I guess your trying to emphasize that adjusting for plausible scenarios also means adjusting so that implausible ones are no longer present?

Reply:
Adjusted. Thanks.